src/pages/RegisterAndLogin/Components/Inputs.tsx

```diff
@@ -134,11 +135,7 @@ const Inputs = ({ page }: { page: string }) => {
               children={open ? <VisibleIcon /> : <InvisibleIcon />}
             />
           </InputGroup>
-          {formErrors.password ? (
-            <Text color="red" as="i">
-              {formErrors.password}
-            </Text>
-          ) : null}
+          <PassRqrmt formData={formData} />
```

Comment:
We have one big mistake. No need to show this section during login. It should be visible during registration.